app/Livewire/Server/Show.php

```
@@ -298,11 +298,36 @@ public function updatedIsMetricsEnabled($value)
         }
     }
 
+    public function updatedIsBuildServer($value)
+    {
+        try {
+            $this->authorize('update', $this->server);
+            if ($value === true && $this->isSentinelEnabled) {
+                $this->isSentinelEnabled = false;
+                $this->isMetricsEnabled = false;
+                $this->isSentinelDebugEnabled = false;
+                StopSentinel::dispatch($this->server);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Actions\Server\StopSentinel' in method 'updatedIsBuildServer'. (undefined)

(StaticAccess)